polynote-server/src/main/scala/polynote/server/repository/NotebookRepository.scala

```diff
@@ -47,7 +47,7 @@ trait NotebookRepository {
     */
   def createNotebook(path: String, maybeContent: Option[String]): RIO[BaseEnv with GlobalEnv, String]
 
-  def renameNotebook(path: String, newPath: String): RIO[BaseEnv with GlobalEnv, String]
+  def copyNotebook(path: String, newPath: String, deletePrevious: Boolean): RIO[BaseEnv with GlobalEnv, String]
```

Comment:
Same here, can we drop the boolean flag? We could either have separate `copy` and `rename` operations in the repository as well, or you just have to call `copy` and then `delete` to do a rename.

Reply:
Actually I think it's better to have a `copy` method and a `move` (or `rename` if you need uniformity) method.